llvm/test/CodeGen/NVPTX/unreachable.ll

```diff
@@ -24,25 +26,30 @@ define void @kernel_func() {
   call void @throw()
 ; CHECK-TRAP-NOT: exit;
 ; CHECK-TRAP: trap;
+
 ; CHECK-NOTRAP-NOT: trap;
 ; CHECK: exit;
+
+; CHECK-BUG-FIXED-NOT: exit;
+; CHECK-BUG-FIXED: trap;
```

Comment:
E.g. what if we have `exit` *after* the `trap`?

Reply:
328e12c94e186dc8474646d96355ce4e928e8943